app/build.gradle

```diff
@@ -79,6 +84,21 @@ dependencies {
     // Preference DataStore
     implementation 'androidx.datastore:datastore-preferences:1.0.0'
 
+    testImplementation 'junit:junit:4.13.2'
+
+    testImplementation 'org.mockito.kotlin:mockito-kotlin:3.2.0'
+
+    testImplementation 'io.mockk:mockk:1.13.2'
+    androidTestImplementation 'io.mockk:mockk-android:1.13.2'
+    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4"
+    androidTestImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.4"
+
+    testImplementation 'androidx.arch.core:core-testing:2.1.0'
+    androidTestImplementation "androidx.arch.core:core-testing:2.1.0"
+    testImplementation "com.squareup.okhttp3:mockwebserver:3.12.0"
+
+    implementation "com.github.bumptech.glide:glide:4.13.2"
+    kapt "com.github.bumptech.glide:compiler:4.13.2"
```

Comment:
Minor, but in terms of organization you could move these implementation and kapt lines above the `testImplementation`s.  You could also group your `testImplementation` lines together and your `androidTestImplementation` lines together as well.